Plugins/Public/balancemagic/Main.cpp

```diff
@@ -25,21 +25,16 @@ typedef unsigned char byte;
 #include <FLHook.h>
 #include <plugin.h>
 #include <math.h>
+#include <unordered_map>
 
 bool UserCmd_SnacClassic(uint iClientID, const wstring &wscCmd, const wstring &wscParam, const wchar_t *usage);
 
 typedef void(*wprintf_fp)(std::wstring format, ...);
 typedef bool(*_UserCmdProc)(uint, const wstring &, const wstring &, const wchar_t*);
 
 struct DamageMultiplier {
-	float fighter;
-	float freighter;
-	float transport;
-	float gunboat;
-	float cruiser;
-	float battlecruiser;
-	float battleship;
-	float solar;
+	float projectileDamage;
+	float classMultipliers[21];
```

Comment:
Correct, classes range from 0 to 19, and we need an extra for Solars.

I suppose this could go above 19, but in the coming patch Haste is pruning the number of used classes, so I would say it's not something worth concerning ourselves about.